manifests/velero.yaml

```diff
@@ -50,6 +50,10 @@ metadata:
   namespace: velero
 spec:
   provider: aws
+  {{ if .velero.config.region }}
```

Comment:
I think make this a full copy of the config e.g.

{{ range $k, $v := .velero.config }} 
$k: "$v"
{{end }}